### PR TITLE
21928 Allow corrections for missing types

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#21928

*Description of changes:*
- app version = 7.3.7
- enabled resume for all correction filing types
- enabled correction for all correction filing types (that are not disabled)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/Dashboard/FilingHistoryList/HeaderActions.vue [hunk 1]
```diff
@@ -189,34 +188,9 @@ export default class HeaderActions extends Mixins(AllowableActionsMixin) {
...
-        }
-        break
-    }
-
-    // correction is not supported for all other filings
-    // eslint-disable-next-line no-console
-    console.log('correctThisFiling(), invalid correction type for filing =', this.filing)
+    // show file correction dialog, which will then route to Edit UI
+    this.setCurrentFiling(filing)
+    this.setFileCorrectionDialog(true)
```
Comment (severinbeauvais) [inline src/components/Dashboard/FilingHistoryList/HeaderActions.vue:193]: The code on lines 134-180, above, allows/prevents correction of various filings. By the time we get here, we can assume that correcting this filing is allowed, so we just do it. I think there is no need for more checks.

File: src/components/Dashboard/TodoList.vue [hunk 2]
```diff
@@ -1928,32 +1922,12 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
...
-            }
-            break
-        }
-        // resuming correction is not supported for all other filings
-        // eslint-disable-next-line no-console
-        console.log('doResumeFiling(), invalid correction type for item =', item)
+      case FilingTypes.CORRECTION: {
+        // resume correction via Edit UI
+        const correctionUrl = `${this.getEditUrl}${this.getIdentifier}/correction/?correction-id=${item.filingId}`
+        navigate(correctionUrl)
```
Comment (severinbeauvais) [inline src/components/Dashboard/TodoList.vue:1928]: As above, if we get this far then we can assume that resuming the correction for this filing type is allowed, so we just do it. No need for more checks.

**Review by JazzarKarim — APPROVED** [review]
👍

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-675-4zmhnegg.web.app

**Review by ArwenQin — APPROVED** [review]
LGTM

**Pull request merged**